clio-dataaccess-model/src/test/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndexSpec.scala

```diff
@@ -9,12 +9,14 @@ class ElasticsearchIndexSpec extends FlatSpec with Matchers with Elastic4sAutoDe
   import com.sksamuel.elastic4s.circe._
 
   it should behave like aV1Index(
-    ElasticsearchIndex[DocumentMock](
+    new ElasticsearchIndex[DocumentMock](
+      "mock",
       ElasticsearchFieldMapper.NumericBooleanDateAndKeywordFields
     )
   )
   it should behave like aV2Index(
-    ElasticsearchIndex[DocumentMock](
+    new ElasticsearchIndex[DocumentMock](
+      "mock_v2",
```

Comment:
Would make this `"mock-v2"` to be consistent with the new patterns of the real indexes